src/tier-types.ts

```diff
@@ -531,6 +535,10 @@ export interface CheckoutRequest {
   trial_days?: number
   cancel_url?: string
   require_billing_address?: boolean
+  tax?: {
```

Comment:
I think this should be its own type, and used also by CheckoutParams

Reply:
I thought we wanted to follow existing pattern? https://github.com/tierrun/node-sdk/pull/17#discussion_r1222353635